BlazarService/src/test/java/com/hubspot/blazar/BlazarServiceTestBase.java

```diff
@@ -3,29 +3,15 @@
 import static org.assertj.core.api.Assertions.fail;
 
 import org.junit.After;
-import org.junit.BeforeClass;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
-import com.google.inject.Guice;
-import com.hubspot.blazar.test.base.service.BlazarTestBase;
+import com.google.inject.Inject;
+import com.hubspot.blazar.test.base.service.DatabaseBackedTest;
 
-public class BlazarServiceTestBase extends BlazarTestBase {
-  private static final Logger LOG = LoggerFactory.getLogger(BlazarServiceTestBase.class);
-
-  @BeforeClass
-  public static void setup() throws Exception {
-    synchronized (injector) {
-      if (injector.get() == null) {
-        injector.set(Guice.createInjector(new BlazarServiceTestModule()));
-        runSql("schema.sql");
-      }
-    }
-  }
+public class BlazarServiceTestBase extends DatabaseBackedTest {
 
   @After
-  public void cleanup() throws Exception {
-    runSql("schema.sql");
+  @Inject
+  public void checkEventBusExceptions() throws Exception {
```

Comment:
Are all services that write in the Db reading/writing in the Event Bus? In other words for which services is this check relevant? Maybe it should be added only in the relevant test classes.